### PR TITLE
initiate_db fixes

### PR DESCRIPTION
**Pull request by autogestion**

### CODE REVIEW

**Comment by autogestion** [comment]
Also, lets move from number id in OrganizationType.ORG_TYPES to alphabetic short identificator, like those, which is now used in test GeoJSON as AMENITY field (like "police", "government"). It will make more clear to create and parse GeoJSON.
Names and set stays still, only id change

File: claim/models.py [hunk 1]
```diff
@@ -158,6 +166,7 @@ def get_json_claims(self):
         # Remove me after release. Not critical.
         claims = Claim.objects.filter(
             organization__in=self.organizations.all())
+        print(self.organizations.all(), claims)
```
Comment (Vegasq) [inline claim/models.py]: Remove debug messages

File: templates/add_page.html [hunk 2]
```diff
@@ -167,6 +168,8 @@
             var claim_text = $('#claim_text').val();
             if (claim_text.length < AddPage.min_claim_text_len) {
                 $('#claim_text').addClass('incorrect');
+                $('#claim_text').val(AddPage.min_claim_text_len + " characters is minimum length of text");
+
```
Comment (Vegasq) [inline templates/add_page.html:173]: I think we should keep original value to let user modify it even if it incorrect so far. Massege can be shown in some div or popup.

File: claim/models.py [hunk 3]
```diff
@@ -33,6 +33,7 @@ class OrganizationType(models.Model):
         ("16", _("Міністерство аграрної політики та продовольства України")),
         ("17", _("Міністерство юстиції України")),
     )
+    # TODO(autogestion) unique=True?
```
Comment (Vegasq) [inline claim/models.py]: Make sence for me

**Pull request merged**